### DISCUSSION TITLE
lesson 6 missing files

### DISCUSSION THREAD

**Discussion by HussamX98** [Cyfrin/foundry-full-course-cu#130] [Q&A] [answered]
I am at the part where he used the code ''' forge init --force .''' and when used, it loaded all the files except "lib" and ".gitmodules". So how should I fix this issue?

**Comment by cromewar** [comment, marked as answer]
Hello @HussamX98 
Can you try using just the name of the project, like:
`forge init projectName` and see if you get the same problem?

**Reply by CodeTivity08** [reply]
Thanks, cromewar.
